Kalman Filter rework

IotaCZE opened the pull request:
Kalman filter page, previously done by Ondrej Franek, needed rework. I have rewritten all of it to be compatible with markdown and also added short summary for KF

IotaCZE committed:
Rewritten everything, so it is readable in Markdown and LaTex. Added summary for KF

peci1 commented on docs/kalman/kalman.md:
```diff
@@ -6,204 +6,217 @@ mathjax: true
+  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+</script>
+{% endraw %}
+
```
The stuff in `raw` block should not be a part of this MD file. If needed, change config in https://github.com/ctu-vras/autonomous-robotics/blob/main/_layouts/default.html .

peci1 commented on docs/kalman/kalman.md:
```diff
@@ -6,204 +6,217 @@ mathjax: true
+{: .no_toc }
+This chapter explains the basics of Bayes filter and of (extended) Kalman filter.
+
+<details open markdown="block">
```
Do not add own TOC. It will be autogenerated.

peci1 reviewed (changes requested):
There are a few changes that need to be done before this PR can be merged. Let me know if you can make and test them.

Thank you,
Martin

peci1 committed:
Remove MathJax scripts and TOC from kalman.md

Removed MathJax configuration scripts and table of contents details from the Kalman filter documentation.
